src/taskProvider.ts

```diff
@@ -0,0 +1,68 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+'use strict';
+
+import * as serverUtils from './omnisharp/utils';
+import * as vscode from './vscodeAdapter';
+
+import { OmniSharpServer } from './omnisharp/server';
+import { TaskRevealKind } from 'vscode';
+
+export function activate(_context: vscode.ExtensionContext, server: OmniSharpServer): vscode.Disposable {
+	if (!vscode.workspace.workspaceFolders) {
```

Comment:
Please use spaces rather than tabs. This is defined the repro's .vscode/settings.json file.

Reply:
not sure why that wasn't respected... fixed.

Reply:
also added a tslint rule

Reply:
Thanks!